src/components/Dashboard/MainContainer/MainContainerUtils.js

```diff
@@ -14,20 +14,20 @@ export const getTaskListAsArray = (snapValue = {}) => {
   return taskList;
 };
 
-export const deleteDragOver = (event) => {
+export const deleteDragByEvent = (event, type) => {
   event.preventDefault();
-  document.getElementById('delete-zone').classList.add('drag-in');
-};
-
-export const deleteDragLeave = (event) => {
-  event.preventDefault();
-  document.getElementById('delete-zone').classList.remove('drag-in');
+  if (type === 'over') {
+    document.getElementById('delete-zone').classList.add('drag-in');
+  } else if (type === 'leave') {
+    document.getElementById('delete-zone').classList.remove('drag-in');
+  }
```

Comment:
`document.getElementById('delete-zone').classList[type === 'over' ? 'add' : 'remove']('drag-in');`